spec/spec_helper.rb

```diff
@@ -60,6 +60,14 @@ def clock_time
     end
   end
 
+  # A directory where SSL pem files exist.
+  def ssl_dir
+    return @ssl_dir if @ssl_dir
+
+    @ssl_dir = ENV['TEST_RUBY_MYSQL2_SSL_DIR'] || '/etc/mysql'
```

Comment:
The name of the `TEST_RUBY_MYSQL2_SSL_DIR` is because there is one environment `RUBY_MYSQL2_LIBMYSQL_DLL` used. So, I referred to the name.

```
$ grep -r RUBY_MYSQL2
README.md:* Environment variable `RUBY_MYSQL2_LIBMYSQL_DLL=C:\path\to\libmysql.dll`
tmp/x86_64-linux/stage/README.md:* Environment variable `RUBY_MYSQL2_LIBMYSQL_DLL=C:\path\to\libmysql.dll`
tmp/x86_64-linux/stage/lib/mysql2.rb:  dll_path = if ENV['RUBY_MYSQL2_LIBMYSQL_DLL']
tmp/x86_64-linux/stage/lib/mysql2.rb:    ENV['RUBY_MYSQL2_LIBMYSQL_DLL']
lib/mysql2.rb:  dll_path = if ENV['RUBY_MYSQL2_LIBMYSQL_DLL']
lib/mysql2.rb:    ENV['RUBY_MYSQL2_LIBMYSQL_DLL']
```